scan.go

```diff
@@ -668,6 +690,7 @@ func internalStartScan() error {
 		BeforeFinish: func() {
 			ui.MarkTrackerAsDone(packageManifestTracker)
 			ui.MarkTrackerAsDone(packageTracker)
+			ui.MarkTrackerAsDone(syncReportTracker)
```

Comment:
Possible `nil` deref here because we are creating `syncReportTracker` conditionally at 

https://github.com/safedep/vet/pull/400/files#diff-37272e63b3a935ee8196994289044044d85351186f29286b458a60fd513183a4R677